src/observability/summarizer.go

```diff
@@ -51,7 +51,12 @@ func extractNetworkInfoFromSystemLog(netLog pb.Log) (string, string, string, str
 				}
 			}
 		}
-	} else if strings.Contains(netLog.Data, "SYS_BIND") {
+	}
+
+	if strings.Contains(netLog.Data, "SYS_BIND") {
+		bindAddress = ip
+	}
+	if strings.Contains(netLog.Data, "SYS_BIND") {
```

Comment:
```suggestion
    	if strings.Contains(netLog.Data, "SYS_BIND") {
	    	bindAddress = ip
	}

```
what is the point of using two different `if` conditions with the same rule?

Reply:
Changed.